### ISSUE TITLE
Styling Refactor on Non Clickable Options of File Menu

### ISSUE THREAD

**Issue by runningonsideprojects** [processing/p5.js-web-editor#4121]
## Issue Description

This specific subissue aims to address the several stylistic issues concerning the non clickable options of the File menu of the editor. 

Namely: 
1. Changing the text color of the non-clickable menu option to adhere to WCAG guidelines
2. Remove the on hover highlight for non-clickable menu options.

Location of the requested changes on the editor: https://editor.p5js.org/ 

## Change Requested

WCAG guidelines [state](https://www.allaccessible.org/blog/color-contrast-accessibility-wcag-guide-2025) that non-text color contrast should be at least 3:1, and small text contrast should be 4.5:1. The foreground color the grayed out text is `#747474`, while the menu background color is `#f0f0f0`. This results in a color contrast ratio of 4.1:1. 

- [ ] The text color should be changed to `#666`
- [ ] Remove the on hover menu highlight + tooltip on non clickable options

## Visuals

Original (grayed out text: `#747474`. Regular text: `#333`) 

<img width="396" height="512" alt="Image" src="https://github.com/user-attachments/assets/80d4a9a4-bfb0-4b11-9b9d-82c5f481868b" />

Text color modifications: 
- Grayed out text: `#666` (`$middle-dark`)
- Regular text: `#1C1C1C`(`$dark`)

<img width="390" height="512" alt="Image" src="https://github.com/user-attachments/assets/cac5a416-00af-4bcf-a872-f5fbbabb4b6f" />

On Hover Tooltip: 

<img width="469" height="512" alt="Image" src="https://github.com/user-attachments/assets/717359d3-cb54-4e35-915a-63750cbb7246" />

Files to change
- `client/…/Header/Nav.jsx`
- `client/…/components/_nav.scss`

PR Criteria
- [ ] Provide screenshots in ur PR of the visual changes implemented
- [ ] Provide QA steps for how reviewers can ensure your changes are made properly

Additional Resources:
 
- [Color Combinations for Accessibility](https://github.com/processing/p5.js-web-editor/wiki/Color-Combinations-for-Accessibility) – tool developed by @izzy-synder that generates custom color palettes for components while adhering to WCAG guidelines and [p5.js](http://p5.js/) styling standardizations. 
- [Zeplin Style Link](https://scene.zeplin.io/project/55f746c54a02e1e50e0632c3) – the p5.js editor’s styling standardizations
- [Preparing a Pull Request](https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/preparing_a_pull_request.md) – how-to guide on creating a Pull Request 
- [Style Guide Documentation](https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/styling.md) (WIP), best-practices guide on implementing styling changes to the codebase.

**Comment by runningonsideprojects** [comment]
@ksen0 i can work on a PR for this issue!

**Comment by parthjadhao01** [comment]
Hi @runningonsideprojects , are you still working on this? I'd love to pick it up if you're no longer able to

**Comment by runningonsideprojects** [comment]
I just wanted to provide an update to my progress on this PR ... 

Currently dealing with an irksome css precedence issue. Im trying to make sure the text of the disabled menu options is `#666 ($middle-dark)` but when you hover there is a glitch where the text is white for a split second and ive been trying to the bottom of it before submitting my PR.

I have solicited advice on the discord from other folks who may some insight into this. im hoping maybe i can hear back from some folks on what to do. 

<img width="1106" height="640" alt="Image" src="https://github.com/user-attachments/assets/107cb600-90bc-47d8-9fdf-ecb09c412c95" />

https://github.com/user-attachments/assets/88767fe0-4544-4c93-ae88-04c022625b94

**Comment by runningonsideprojects** [comment]
> I just wanted to provide an update to my progress on this PR ...
> 
> Currently dealing with an irksome css precedence issue. Im trying to make sure the text of the disabled menu options is `#666 ($middle-dark)` but when you hover there is a glitch where the text is white for a split second and ive been trying to the bottom of it before submitting my PR.
> 
> I have solicited advice on the discord from other folks who may some insight into this. im hoping maybe i can hear back from some folks on what to do.
> <img alt="Image" width="1106" height="640" src="https://private-user-images.githubusercontent.com/227033681/600495697-107cb600-90bc-47d8-9fdf-ecb09c412c95.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODA1OTU0NDUsIm5iZiI6MTc4MDU5NTE0NSwicGF0aCI6Ii8yMjcwMzM2ODEvNjAwNDk1Njk3LTEwN2NiNjAwLTkwYmMtNDdkOC05ZmRmLWVjYjA5YzQxMmM5NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwNjA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDYwNFQxNzQ1NDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03ZjljYzAyYzk2YzNkYTVhOWViNmU3NjM4YjFjMmNlODA2OGY4ZTA3ZDc2M2U1NTA1MDdkNjI5ZjEzMWRmYWI2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZyZXNwb25zZS1jb250ZW50LXR5cGU9aW1hZ2UlMkZwbmcifQ.McwENz1ugw5vyp47C-8ndsdKxm1F9pu4cVg_K46GWd8">
> Screen.Recording.2026-05-30.at.10.03.32.AM.mov

[PR](https://github.com/processing/p5.js-web-editor/pull/4145) is out!!!!!!!! and this has been fixed finally lol
